build.gradle

```diff
@@ -588,7 +588,8 @@ dependencies {
     compile 'org.grails.plugins:browser-detection:3.4.0'
 
     // enable `import grails.plugin.cache.Cacheable`
-    implementation 'org.grails.plugins:cache'
+    implementation 'org.grails.plugins:cache:4.0.0'
```

Comment:
Was there a reason for choosing 4.0.0 instead of 4.0.3?

Reply:
I was trying a lot of versions, and I forgot to change it back to 4.0.3. Already bumped up